TS02-Convert-to-Date-Time-Class-POSIX.Rmd

```diff
@@ -86,22 +87,12 @@ library(lubridate)  #work with dates
 #set working directory to ensure R can find the file we wish to import
 #setwd("working-dir-path-here")
 
```

Comment:
sounds perfect. As long as the data are loaded in this lesson as well, we are good! thank you! :)